test/suite/test_tiered10.py

```diff
@@ -26,50 +26,80 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
+from helper_tiered import get_auth_token, get_bucket1_name, get_bucket1_region
+from helper_tiered import generate_s3_prefix
+from wtscenario import make_scenarios
 import os, wiredtiger, wttest
 StorageSource = wiredtiger.StorageSource  # easy access to constants
 
 # test_tiered10.py
 #    Test tiered storage with simultaneous connections using different
 # prefixes to the same bucket directory but different local databases.
 class test_tiered10(wttest.WiredTigerTestCase):
+    storage_sources = [
+        ('local', dict(ss_name = 'local_store',
+            auth_token = get_auth_token('local_store'),
+            bucket = get_bucket1_name('local_store'),
+            bucket_region = get_bucket1_region('local_store'),
+            prefix1 = '1_',
+            prefix2 = '2_')),
+        # WT-8896 - S3 extension doesnt gets stuck at initializing if more than one simultaneous
```

Comment:
Same comment as before. I think `FIXME` is needed and same suggested wording change.